scripts/monitor_homeassistant_philips.ps1

```
@@ -0,0 +1,175 @@
+param(
+    [string]$HomeAssistantUrl = "http://192.168.1.153:8123",
+    [string]$Username = $env:HA_USERNAME,
+    [securestring]$Password,
+    [string]$PasswordPlain = $env:HA_PASSWORD,
```

PSScriptAnalyzer (1.25.0)

[warning] Parameter '$PasswordPlain' should not use String type but either SecureString or PSCredential, otherwise it increases the chance to expose this sensitive information.

Suggested fix: Set $PasswordPlain type to SecureString; Set $PasswordPlain type to PSCredential

(PSAvoidUsingPlainTextForPassword)